IAMOTZ/whois
# Restructure codebase

### What does this PR do
- move application codes into an `src` directory
- add a start and build script to `package.json`
- update `babel` and babel-related packages to version 7.x
- update `babel` configuration

### How should it manually tested
- Clone and set up the project
- Check out to the branch containing the changes in this PR
- Run `npm test` to execute the tests. All the tests should pass.
- Run `npm run start:dev` to start the app in dev mode. The app should start fine.
- Run `npm start` to start the app in production mode. The app should first build its production version into a `build` folder and then start fine.
